src/content/docs/validator/external-chains/hyperliquid.mdx

```diff
@@ -116,6 +118,10 @@ After a few minutes, you should see lines that look like this. This means your n
 2025-05-26T06:59:40.688Z WARN >>> hl-node @@ applied block 607619600
 ```
 
+> **Note:** The network can generate up to 100 GB of logs per day, so it is strongly recommended to archive or delete old files regularly.  
+> For more details, refer to the [official documentation](https://github.com/hyperliquid-dex/node?tab=readme-ov-file#reading-l1-data).  
+> You can also configure [`logrotate`](https://manpages.ubuntu.com/manpages/focal/man8/logrotate.8.html) to manage log file rotation automatically.
```

Comment:
can you add instructions on how to configure it (exact cmds to use)? otherwise everyone needs to look into it themselves